sources/packages/web/src/store/modules/student/student.ts

```diff
@@ -0,0 +1,53 @@
+
+export class StudentState {
+
+     profile!: StudentProfile;
+}
+
+export class StudentProfile {
+    name?: string;
+    age?: string;
+    birthdate?: string;
+    displayName?: string;
+    email?: string;
+    emailVerified?: string;
+    familyName?: string;
+    gender?: string;
+    givenName?: string; 
+    middleName?:string;
+}
+
+const initialState: StudentState = {
+    profile: new StudentProfile()
+}
+
+export const student = {
+    namespaced: true,
+    state : initialState,
+   
+    mutations: {
+        SET_STUDENT_PROFILE_DATA(state:StudentState, parsedToken:any){
+            //console.log(`mutation p${parsedToken.age}`)                        
```

Comment:
Please consider removing the commented lines.

Reply:
done